api/episodesByFeedId/[episodesByFeedId].js

```diff
@@ -0,0 +1,10 @@
+import fetchDataFromDB from '../fetchDataFromDB.js'
+
+export default async function podcastByFeedId(req, res) {
+  const feedId = req.query.episodesByFeedId
+  const url = '/episodes/byfeedid?id=' + feedId
```

Comment:
Just personal preference for readability
```suggestion
  const url = `/episodes/byfeedid?id=${feedId}`
```

Reply:
Yes i changed it everywhere now. Much better thanks :)